crates/brioche/src/migrate_registry_to_cache.rs

```diff
@@ -481,9 +481,7 @@ async fn save_project(
     let mut git_refs = BTreeMap::new();
     for (static_, output) in project.statics.values().flatten() {
         match static_ {
-            StaticQuery::Include(_) | StaticQuery::Glob { .. } => {
-                continue;
-            }
+            StaticQuery::Include(_) | StaticQuery::Glob { .. } => {}
```

Comment:
There is no need for a continue statement here, since there is nothing after the match (https://rust-lang.github.io/rust-clippy/rust-1.86.0/index.html#needless_continue)